docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,55 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[post-upgrade-reqs-8.0.0]]
+==== Post upgrade requirements
+* Changes to the indicator match rule's default threat indicator path (<<rule-ui-advanced-params, *Indicator prefix override*>>) might require you to update existing rules, or create new ones, after upgrading to {stack} version 8.0.0. Be mindful of the following ({pull}118821[#118821]):
+** If an indicator match rule's default threat indicator path was not defined before upgrading, it will receive the default value `threatintel.indicator` during the upgrade process. This allows the rule to continue using indicator data ingested by {filebeat} version 7.x. If a custom value was defined before the upgrade, the value remains the same.
+** If an existing indicator match rule was configured to use threat indicator indices generated from {filebeat} version 7.x, updating the default threat indicator path to `threat.indicator` after you upgrade to {stack} version 8.0.0 and {agent} or {filebeat} version 8.0.0 configures the rule to use threat indicator indices generated by {agent} or {filebeat} version 8.0.0.
```

Comment:
Ignore if you feel this loses necessary specificity, but I was trying to avoid repetition:
```suggestion
** If an existing indicator match rule was configured to use threat indicator indices generated from {filebeat} version 7.x, updating the default threat indicator path to `threat.indicator` after you upgrade to {stack} version 8.0.0 and {agent} or {filebeat} version 8.0.0 configures the rule to use threat indicator indices generated by those later versions.
```

Reply:
I don't think this change detracts from the intended message. It's also easier to read, so no objections to your suggestion.